src/main/java/com/fasterxml/jackson/databind/ObjectMapper.java

```diff
@@ -2552,6 +2552,11 @@ public boolean isEnabled(SerializationFeature f) {
     /**
      * Method for changing state of an on/off serialization feature for
      * this object mapper.
+     * <p>
+     * <strong>WARNING: This method will be removed in Jackson 3.0 due to unsafe usage.</strong>
```

Comment:
I think that this removal should either go along with reference to `MapperBuilder` method to use instead, or, we just remove removal notice and keep next 3 lines that are important. After all, usage in 2.x is legal as long as it occurs before use.